src/test/java/org/tikv/integration/IntegrationTest.java

```diff
@@ -0,0 +1,146 @@
+package org.tikv.integration;
+
+import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.fail;
+
+import com.google.protobuf.ByteString;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Before;
+import org.tikv.common.TiConfiguration;
+import org.tikv.common.TiSession;
+import org.tikv.common.exception.RegionException;
+import org.tikv.common.region.RegionStoreClient;
+import org.tikv.common.region.TiRegion;
+import org.tikv.common.util.BackOffFunction;
+import org.tikv.common.util.BackOffer;
+import org.tikv.common.util.ConcreteBackOffer;
+import org.tikv.kvproto.Kvrpcpb;
+
+public class IntegrationTest {
+  static final int DEFAULT_TTL = 10;
+  private static final String DEFAULT_PD_ADDR = "127.0.0.1:2379";
+  private TiSession session;
+  RegionStoreClient.RegionStoreClientBuilder builder;
+
+  @Before
+  public void setUp() {
+    TiConfiguration conf = TiConfiguration.createDefault(getPdAddr());
```

Comment:
```suggestion
    TiConfiguration conf = TiConfiguration.createDefault();
```

we can already use `-Dtikv.pd.addresses` to set the pd address.

Reply:
fixed